code/game/objects/items/devices/PDA/virus_cart.dm

```diff
@@ -68,7 +68,7 @@
 		if(target.cartridge)
 			difficulty += BitCount(target.cartridge.access&(CART_MEDICAL | CART_SECURITY | CART_ENGINE | CART_CLOWN | CART_JANITOR | CART_MANIFEST))
 			if(target.cartridge.access & CART_MANIFEST)
-				difficulty++ //if cartridge has manifest access it has extra snowflake difficulty
+				difficulty += 4 //if cartridge has manifest access it has extra snowflake difficulty
 			else
 				difficulty += 2
```

Comment:
```suggestion
				difficulty ++ 4 //if cartridge has manifest access it has extra snowflake difficulty
```

Reply:
giving cartridges with manifest access +4 to their difficulty rating (actually a +5 because CART_MANIFEST is in the list of cartridge accesses that are counted up to determine difficulty) would make most PDAs with a security or head of staff cartridge nigh-impossible to detonate with a detomatix. Remember, each point of difficulty adds 15% to the chance for the detomatix attempt to fail, and each detomatix only has 4 charges.

Reply:
Ah, sorry I misunderstood what that was referring to. Fixing that now

Reply:
I did what Fikou had suggested, I believe it might be fixed now